client/extensions/woocommerce/woocommerce-services/state/shipping-label/reducer.js

```diff
@@ -706,7 +903,7 @@ reducers[ WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SAVE_CUSTOMS ] = state => {
 			...state.form,
 			customs: {
 				...state.form.customs,
-				submitted: true,
+				ignoreTariffNumberValidation: {},
```

Comment:
I'm having a hard time understanding what `WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SAVE_CUSTOMS` means exactly. In the dummy PR, it meant that the customs step was submitted ( setting `submitted: true` );

I'm asking so that it can potentially be renamed to something that more immediately describes its effects.

Here, it looks to be resetting the value of `ignoreTariffNumberValidation` to an empty object - presumably because when the customs step is opened again we'll want to recalculate whether we're ignoring the tariff number validation.

I'm not sure my interpretation is exactly correct, because the initialization of the customs state have a non-empty object value for `ignoreTariffNumberValidation` ([code ref](https://github.com/Automattic/wp-calypso/pull/24974/files#diff-45c5247f05768aa908f6e59f26bd6d3dR120)).

So - what I'm asking is - what does this reducer really mean in the bigger context?

Context: This reducer is for `WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SAVE_CUSTOMS`.

Reply:
> Here, it looks to be resetting the value of ignoreTariffNumberValidation to an empty object - presumably because when the customs step is opened again we'll want to recalculate whether we're ignoring the tariff number validation.

The `ignoreTariffNumberValidation` is set to an empty object, so it means "all the tariff numbers must be validated". This triggers a full validation of the step. It's the same, conceptually, as marking all the fields "dirty" so their contents are checked. This is because we don't want to display errors before the user interacts with a given field, but we still need to validate the entire step when the user clicks the "Save" button.